### PR TITLE
Merge recent changes to master

### PR DESCRIPTION
**Pull request by nibinqtl**

### CODE REVIEW

File: docs/edge-logic/supported-directives.md [hunk 1]
```diff
@@ -112,7 +112,7 @@ This directive enables proxying the WebSocket protocol. The client has to make s
 
 <span class="badge">advanced</span>
 
-Defines the URI that will be shown for the specified errors. No change to the public version. 
+Defines the URI that will be shown for the specified errors codes. No change to the public version. We configured [`proxy_intercept_errors on`](http://nginx.org/en/docs/http/ngx_http_proxy_module.html#proxy_intercept_errors) to make it work for error codes returned from the origin.
```
Comment (burtonquantil) [inline docs/edge-logic/supported-directives.md:115]: "errors codes" should be "error codes".

File: docs/edge-logic/supported-directives.md [hunk 2]
```diff
@@ -598,6 +592,12 @@ Defines an expression for which the MD5 hash value will be computed and compared
 
 Defines a secret word used to check authenticity of requested links. No change to the public version.
 
+### [`set`](http://nginx.org/en/docs/http/ngx_http_rewrite_module.html#set)
+
+<span class="badge">standard</span>
+
+Assigns a value to the specified variable. No change to the public version. In particular, the cache key [can be customized](/docs/edge-logic/faq.md#how-do-you-include-query-parameters-andor-request-headers-in-the-cache-key) by assigning value to the `$cache_misc` variable.
```
Comment (burtonquantil) [inline docs/edge-logic/supported-directives.md:599]: "by assigning value" should be "by assigning a value" .

File: docs/intro.md [hunk 3]
```diff
@@ -1,23 +1,23 @@
 # Welcome to CDN360
 
-CDN360 leads the evolution of content delivery networks (CDNs) by accelerating, securing, scaling, and delivering content to website visitors worldwide, regardless of their location. With points of presence on every continent, visitor requests are routed to edge servers based on factors such as proximity and resource/content availability.
+CDN360 is leading the next evolution of content delivery networks (CDN). 
 
-By keeping content close to website visitors, utilizing intelligent request routing, and employing high-performance computing, storage, and network resources, CDN360:
+CDN360 is designed to accelerate and deliver your applications to users worldwide, regardless of their location. It boosts content delivery, improves load times, and many more clear-cut advantages to deliver an unparalleled user experience.
```
Comment (burtonquantil) [inline docs/intro.md:5]: The sentence beginning with "It boosts" is not grammatically correct.    @marcfishelquantil 
I would add the word "includes" before "many more clear-cut advantages".

File: docs/edge-logic/supported-directives.md (hunk 2 above)
Comment (nibinqtl) [reply docs/edge-logic/supported-directives.md]: fixed

File: docs/edge-logic/supported-directives.md (hunk 1 above)
Comment (nibinqtl) [reply docs/edge-logic/supported-directives.md]: fixed

File: docs/intro.md (hunk 3 above)
Comment (marcfishelquantil) [reply docs/intro.md:5]: Yes, or I would propose saying "delivers" or "offers."

Comment (burtonquantil) [reply docs/intro.md:5]: "offers" is goo.

Comment (nibinqtl) [reply docs/intro.md]: I just made the change. @burtonquantil Any other concerns?

**Comment by burtonquantil** [comment]
OK to merge. If we notice anything else, we can file a new bug.

**Pull request merged**